cvmfs/cvmfs_server

```diff
@@ -787,16 +787,16 @@ is_stratum1() {
 
 # checks if a given repository is replicable
 #
-# @param name   the repository name to be checked
+# @param name   the repository name or URL to be checked
 # @return       0 if it is a stratum0 repository and replicable
 is_master_replica() {
   local name=$1
   local is_master_replica
 
-  load_repo_config $name
   if [ $(echo $name | cut --bytes=1-7) = "http://" ]; then
     is_master_replica=$(cvmfs_swissknife info -r $name -m $(get_follow_http_redirects_flag) 2>/dev/null)
```

Comment:
It looks like we do not know the repository name necessarily and thus the load_repo_config cannot be called. This means that
$(get_follow_http_redirects_flag)
will never return anything. Should we remove it or have "-L" as a default (to follow the redirects)?